plugins/web-plugin/src/main/java/ru/sbtqa/tag/pagefactory/web/configure/WebDriverManagerConfigurator.java

```diff
@@ -17,20 +16,19 @@
 import java.util.List;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
+import org.aeonbits.owner.ConfigFactory;
 import static org.apache.commons.lang3.SystemUtils.IS_OS_LINUX;
 import static org.apache.commons.lang3.SystemUtils.IS_OS_MAC;
 import static org.apache.commons.lang3.SystemUtils.IS_OS_WINDOWS;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import static ru.sbtqa.tag.pagefactory.support.BrowserType.IE;
-import ru.sbtqa.tag.pagefactory.support.properties.Configuration;
-import ru.sbtqa.tag.pagefactory.support.properties.Properties;
+import ru.sbtqa.tag.pagefactory.web.properties.WebConfiguration;
+import ru.sbtqa.tag.pagefactory.web.support.BrowserName;
 
 public class WebDriverManagerConfigurator {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)